src/processors/models/processorsManager.js

```diff
@@ -79,6 +83,31 @@ function generateProcessorNameAlreadyExistsError() {
     return error;
 }
 
+function generateUnprocessableEntityError(message) {
```

Comment:
we have 4 `generateErrorFunctions` in this file. Refactor to `./errors.js` file in the same directory to organize it a bit more